taxonomy-editor-frontend/src/pages/startproject/index.jsx

```diff
@@ -0,0 +1,113 @@
+import { Typography, Box, Grid, TextField, Stack, Autocomplete, Snackbar, Alert, CircularProgress, Button } from "@mui/material";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { TAXONOMY_NAMES } from "../../constants";
+import { createBaseURL } from "../editentry/createURL";
+import { toSnakeCase } from "../editentry/interConvertNames";
+
+const StartProject = () => {
+    const [branchName, setBranchName] = useState("")
+    const [taxonomyName, setTaxonomyName] = useState(null)
+    const [description, setDescription] = useState("");
+    const [loading, setLoading] = useState(false); 
+    const [errorMessage, setErrorMessage] = useState(null);
+    const navigate = useNavigate();
+    
+    const handleSubmit = () => {
+        const baseUrl = createBaseURL(taxonomyName, branchName)
+        setLoading(true);
+        const dataToBeSent = {'description' : description};
+        fetch(baseUrl+'import', {
+            method : 'POST',
+            headers: {"Content-Type" : "application/json"},
+            body: JSON.stringify(dataToBeSent)
+        }).then(async (response) => {
+            const responseBody = await response.json();
+            if (!response.ok) {
+                throw Error(responseBody.detail)
+            }
+            navigate(`/${taxonomyName}/${branchName}/entry`)
+        }).catch((detail) => {
+            setErrorMessage(detail.message); setLoading(false);
```

Comment:
mmm ,  we need to check that the error has a `.message`. Also not 100% sure we want to show whatever is returned from the backend (like what happens with localization, and how can we ensure that this error has the information that is user friendly), but would need to see what errors are returned (like how different they are and if it is worth to handle it in this way). Guess for now I'd tend to think is safer not to have it like this and just return something simple as `unable to save`

Reply:
The API does return meaningful messages currently. But, it should be safer to keep a simple message.

Reply:
cool, yep, another issue is talked in the past if we want to have internationalization (translation) which will add the need to get a localized (translated) string from the API. I think is just fine to show a generic message.